src/main/java/teammates/ui/template/CourseEditInstructorPanel.java

```diff
@@ -45,12 +46,17 @@ public CourseEditInstructorPanel(int instructorToShowIndex, int instructorIndex,
             if (instructor.privileges.numberOfSectionsSpecial() >= sectionNames.size()) {
                 style = "display: none;";
             }
-            String onClick = "showTuneSectionPermissionsDiv(" + index + ", "
-                             + firstBlankSectionRowIndex + ")";
             String content = "Give different permissions for a specific section";
-            String id = "addSectionLevelForInstructor" + index;
-            addSectionLevelForInstructorButton = createButton(content, "small", id, "javascript:;",
-                                                              style, onClick);
+
+            Map<String, String> attributes = new HashMap<>();
+            attributes.put("class", "small addSectionLevelForInstructor");
+            attributes.put("id", "addSectionLevelForInstructor" + index);
```

Comment:
The ID and class should be `like-this` instead of `likeThis`. I understand the ID was `likeThis` before and you can choose to retain it for consistency, but the class should follow `this-convention`.